src/main/java/com/mide/gangsaeng/board/BoardMapper.java

```diff
@@ -5,4 +5,5 @@
 @Mapper
 public interface BoardMapper {
     void write(BoardRequest request);
+    Board read(Long id);
```

Comment:
Why we should get id by wrapper class rather than primitive type?

Reply:
Wrapper class can be `null` , but primitive type can't. It can represent missing or undefined values in databases.

Reply:
Then we should validate in controller layer?

Reply:
Add validation in controller (using validation library) for bad request.

Reply:
Then what happen if user give us bad parameter?

And why don't you also handle read method? I want to avoid wrapper class in storage/service layer.

Reply:
> Then what happen if user give us bad parameter?

I think if there is not proper exception handling, data of incorrect types will be stored in storage.

> And why don't you also handle read method? I want to avoid wrapper class in storage/service layer.

I fixed it.